src/io/flutter/FlutterInitializer.java

```diff
@@ -174,7 +155,7 @@ public void runActivity(@NotNull Project project) {
 
       final Notification notification = new Notification(
         Analytics.GROUP_DISPLAY_ID,
-        FlutterBundle.message("flutter.analytics.notification.title"),
+        "Welcome to Flutter!",
```

Comment:
Phil, Brian S. and I had a discussion a few months ago, and generally agreed that the overhead of maintaining the i18n file doesn't really pay off until we start actually localizing the tool. So, it's currently dealer's choice whether each PR choose to i18n strings, or put them in the code raw.

For this case, I thought the shorter string was a bit clearer -